docs/snapcraft-advanced-features.md

```diff
@@ -127,6 +127,25 @@ will be built after `libpipeline`. Especially if you need specific
 functionality during a build or as part of checks during the `stage` phase,
 this will be handy.
 
+If any part built through after needs to explicitly access assets in the
```

Comment:
"built through after" is hard to read.
Maybe: If a part built using the `after` keyword needs ...